python/taichi/misc/gui.py

```diff
@@ -34,14 +34,18 @@ class Event:
     PRESS = ti_core.KeyEvent.EType.Press
     RELEASE = ti_core.KeyEvent.EType.Release
 
-    def __init__(self, name='Taichi', res=512, background_color=0x0):
+    def __init__(self,
+                 name='Taichi',
+                 show_gui=True,
+                 res=512,
```

Comment:
```suggestion
                 res=512,
                 show_gui=True,
```
Some examples wil break by the argument order.